### PR TITLE
Frontend 1868 - Render Dev Center Content in Synapse Layout

### PR DESCRIPTION
**Pull request by langlk**
Resolves [FRONTEND-1868](https://algorithmia.atlassian.net/browse/FRONTEND-1868)

**Things this does do:**
* Renders all pages in the main-page section of synapse layout

**Things this does not do (coming later):**
* Format content nicely within the main page section
* Include side menu links

### CODE REVIEW

**Comment by bnlb** [comment]
Code changes look good, though the styles appears a bit off. I ran git submodule update and rebuilt Synapse, and I'm seeing this. 
Screen Shot 2019-11-01 at 8 41 41 AM

If this is expected as it's a WIP branch and this will be addressed in later branches, then I'm happy to approve.

**Comment by andryanmiller** [comment]
> If this is expected as it's a WIP branch and this will be addressed in later branches, then I'm happy to approve.

That's correct–proper Synapse styling is coming in [this ticket](https://algorithmia.atlassian.net/browse/FRONTEND-1872). Scope of this work just to get the basic layout in place so we could start parallelizing tasks.

File: _layouts/synapse-default.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  {% else %}
+    {% assign webserver = "https://algorithmia.com" %}
+  {% endif %}
+</head>
+
+<body>
+  <div class="syn-page syn-page-drawer-navigation">
+    <header class="syn-page-header syn-header-brand">
+      <div class="syn-page-header-content">
+        <button onclick="['nav-open', 'nav-collapsed'].forEach((a) => document.querySelector('.syn-page-header').classList.toggle(a))" class="syn-btn text icon rounded grow theme-light no-shadow" data-v-3352aef0="" data-v-19ae9fc3="">
```
Comment (andryanmiller) [inline _layouts/synapse-default.html]: :bulb: Going to create a ticket to ensure we bring over the "persistent close/open state" for the menu.

File: _layouts/synapse-default.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+          <svg width="1rem" height="1rem" viewBox="0 0 12 12">
+            <path fill="currentColor" d="M0,2 L12,2 L12,3.344 L0,3.344 L0,2 Z M0,6.67 L0,5.33 L12,5.33 L12,6.67 L0,6.67 Z M0,10 L0,8.656 L12,8.656 L12,10 L0,10 Z"/>
+          </svg>
+        </button>
+      </div>
+      <div class="syn-page-header-nav" onclick="['nav-open'].forEach((a) => document.querySelector('.syn-page-header').classList.toggle(a))">
```
Comment (andryanmiller) [inline _layouts/synapse-default.html]: :bulb: Do we need this `onclick` handler at the moment, or is it something more applicable to a later ticket?

Comment (langlk) [reply _layouts/synapse-default.html]: That's what handles opening/closing the nav

**Pull request merged**